package.json

```diff
@@ -9,6 +9,9 @@
     "url": "https://github.com/Automattic/wp-calypso.git"
   },
   "main": "index.js",
+  "bin": {
+    "wp-calypso-sdk": "./bin/create-scripts/block.js"
```

Comment:
We've been talking about using `calypso-gutenberg-sdk build-block` (see p4TIVU-93I-p2), but also `calypso-gutenberg-sdk` will suffice since that's the single script we have for now.

Reply:
good catch on the name.

also, I wouldn't worry about trying to shorten it for now - we can be safe being verbose in the beginning and simplifying as time goes on.

Reply:
Made it work as `calypso-gutenberg-sdk build-block ...` 🎉